webapp/src/containers/createIncident/callerPanel.tsx

```diff
@@ -47,6 +48,7 @@ const CallerPanel = (({
     const handleCallerSelection = (callerSuggestion: Record<string, string> | null) => {
         setAutoSuggestValue(callerSuggestion?.userName || '');
         setCaller(callerSuggestion?.userId || null);
+        setAutoSuggestDefaultValue(callerSuggestion || {});
```

Comment:
```suggestion
        setAutoSuggestDefaultValue(callerSuggestion ?? {});
```
Shouldn't we use this here?